Can't import absolute URLs, e.g. from CDN

# 🐛 bug report

`import('https://...cdn_path.../module.min.js')` fails with:

> Only absolute URLs are supported
> Failed to install https

The code works just fine without Parcel when I open `index.html` in Chrome.

What am I doing wrong? Failing to load CDN libraries seems like a blatant omission.

## 🎛 Configuration (.babelrc, package.json, cli command)

Zero config.

## 🤔 Expected Behavior

Server launches.

## 😯 Current Behavior

```
$ npx parcel index.html
Server running at http://localhost:1234 
npm ERR! Only absolute URLs are supported

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dandv/.npm/_logs/2018-12-04T20_24_06_209Z-debug.log
🚨  /home/dandv/prg/importcdn/index.js:1:7: Failed to install https:.
> 1 | import('https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js').then(lodash => {
    |       ^
  2 |   console.log(_.VERSION);
  3 | });
  4 | 
```

## 🔦 Context

I'm trying to load from CDN a bunch of libraries I don't want to host myself.

## 💻 Code Sample

**index.html**:
```html
<html>
<body>
<script src="index.js"></script>
</body>
</html>
```
**index.js**:
```js
import('https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js').then(lodash => {
  console.log(_.VERSION);
});
```

## 🌍 Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.10.3
| Node             | 11.3.0
| npm/Yarn         | 6.4.1
| Operating System | Ubuntu Linux 16.04

--- a/packages/core/parcel-bundler/src/visitors/dependencies.js
+++ b/packages/core/parcel-bundler/src/visitors/dependencies.js
@@ -54,6 +54,8 @@ module.exports = {
       types.isStringLiteral(args[0]);
 
     if (isDynamicImport) {
+      if (isURL(args[0].value)) return;
+
       asset.addDependency('_bundle_loader');
       addDependency(asset, args[0], {dynamic: true});
 

--- a/packages/core/parcel-bundler/test/integration/dynamic-external/index.js
+++ b/packages/core/parcel-bundler/test/integration/dynamic-external/index.js
@@ -1,0 +1,5 @@
+const lodash = import('https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js');
+
+export default function () {
+  return lodash.then(() => _);
+};

--- a/packages/core/parcel-bundler/test/javascript.js
+++ b/packages/core/parcel-bundler/test/javascript.js
@@ -342,6 +342,25 @@ describe('javascript', function() {
     assert.equal(await output(), 3);
   });
 
+  it('Should not run parcel over external modules', async function() {
+    let b = await bundle(
+      path.join(__dirname, '/integration/dynamic-external/index.js'),
+      {
+        target: 'browser'
+      }
+    );
+
+    await assertBundleTree(b, {
+      name: 'index.js',
+      assets: ['index.js'],
+      childBundles: [
+        {
+          type: 'map'
+        }
+      ]
+    });
+  });
+
   it('should support bundling workers', async function() {
     let b = await bundle(path.join(__dirname, '/integration/workers/index.js'));
 

